migrate/20230825_add_sshable_user.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:sshable) do
+      add_column :unixuser, :text, collate: '"C"', null: false, default: "rhizome"
```

Comment:
vm has `unix_user`, I think it's good to be consistent
```suggestion
      add_column :unix_user, :text, collate: '"C"', null: false, default: "rhizome"
```